api_end_end_testing/src/main/java/tds/user/api/model/RoleAssociation.java

```diff
@@ -0,0 +1,42 @@
+package tds.user.api.model;
```

Comment:
Okay. Followed standard Maven directory layout where test directories miror src directories. But will change if majority prefers.  https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html